.github/workflows/snl-ci.yml

```diff
@@ -0,0 +1,63 @@
+name: SNL-CI
+
+on: 
+  pull_request:
+    paths-ignore:
+    - '**/*.rst'
+    - '**/*.md'
+    - '**/requirements.txt'
+    - '**/*.py'
+    - 'docs/**'
+    types: [ opened, reopened, synchronize ]
+
+permissions:
+  contents: none
+
+# Cancels any in progress 'workflow' associated with this PR
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  :CUDA_12_2_CPP20
+    name: SNL_CUDA_NVCC_12_2_CPP20
+    runs-on: [snl-kk-env-cuda-12.2.0-gcc-11.3.0-latest]
+    
+    steps:
+      - name: checkout_kokkos
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+        with:
+          repository: kokkos/kokkos
+          ref: ${{ github.base_ref }}
+          path: kokkos
+
+      - name: configure_kokkos
+        run: |
+          mkdir -p kokkos/{build,install}
```

Comment:
Why do you create the install directory ahead of time?
Remember you can/should use `cmake -B build <options>...`